src/modules/add-site/components/blueprints.tsx

```diff
@@ -183,6 +216,11 @@ export default function AddSiteBlueprint( {
 
 				onFileBlueprintSelect( fileBlueprint );
 			} catch ( error ) {
+				if ( error instanceof SyntaxError ) {
+					setValidationError( __( 'Invalid JSON file. Please check the file format.' ) );
```

Comment:
Let's print details of teh error, to be specific and help developers:
<img width="1030" height="83" alt="Screenshot 2025-08-28 at 14 29 36" src="https://github.com/user-attachments/assets/cd228fe2-04b9-437d-8180-39d09ac8b993" />

```suggestion
					setValidationError(
						sprintf(
							// translators: %s is error message of the JSON parsing error
							__( 'Invalid JSON format: %s' ),
							error.message
						)
					);
```